sphinx/Makefile

```diff
@@ -26,6 +26,7 @@ all: html
 
 clean:
 	-rm -rf $(BUILDDIR)/*
+	-rm -rf source/docs/gallery/*
```

Comment:
Some day I would love to find a way to put the generated `.rst` files under `_build` but this will do for now.